### ISSUE TITLE
Error when no .env file

### ISSUE THREAD

**Issue by cbenz** [Andarist/rollup-plugin-dotenv#3]
When I use this Rollup plugin with no `.env` file I get this error in Rollup output:

```
TypeError: functionValues[match[1]] is not a function
```

PS: this issue has been talked about here: https://github.com/ionic-team/stencil/issues/2173

**Comment by Andarist** [comment]
I've tried a bunch of stuff and I don't know how to break the current logic. Could you prepare repro case and list the exact repro steps?

**Comment by cbenz** [comment]
Thanks for your quick reply. Well, I've tried to do a minimal repo but did not achieve to. I've got a repo with a small frontend application [here](https://gitlab.com/francilin/francilin-carto/) that shows the problem, but I feel sorry not being able to reduce it to a minimal proof of concept, so I now have a doubt that the issue comes from rollup-plugin-dotenv.

Anyway here are the steps to reproduce:

```bash
git clone https://gitlab.com/francilin/francilin-carto.git
cd francilin-carto
npm install
npm run build:app
```

Note that there is a `.env.example` that *should* be copied to `.env`, but let's not do it to show the problem.

The output is:

```
> francilin-carto@0.0.1 build:app /home/cbenz/Dev/francilin/francilin-carto
> rollup -c

src/main.js → public/build/bundle.js...
[!] (plugin dotenv) TypeError: functionValues[match[1]] is not a function
src/main.js
TypeError: functionValues[match[1]] is not a function
    at Object.transform (/home/cbenz/Dev/francilin/francilin-carto/node_modules/rollup-plugin-replace/dist/rollup-plugin-replace.cjs.js:75:50)
    at /home/cbenz/Dev/francilin/francilin-carto/node_modules/rollup/dist/shared/node-entry.js:13117:25
```

I tried to do `ndb npm run build:app` but it did not help me to dive into the stack trace.

Feel free to close the issue if you think it's related to my setup rather than rollup-plugin-dotenv.

**Comment by Andarist** [comment]
Thank you for the repro case! I've investigated this and it seems that indeed v0.2 has been suffering from a bug here as it was depending on the outdated `rollup-plugin-replace` which has a bug. A week ago I've released v0.3 which has switched to using migrated `@rollup/plugin-replace` which doesn't suffer from this. Could you just upgrade this package?

**Comment by cbenz** [comment]
OK cool! That explains why the new test project did not have the bug: it installed v0.3.

I confirm that the bug is fixed with v0.3, thanks!
